code42.json

```diff
@@ -1322,26 +1355,6 @@
                     "column_name": "Description",
                     "column_order": 5
                 },
-                {
```

Comment:
I moved parameters to the tops of the outputs list in some cases.

No parameter was harmed in the making of this PR.